<launch>
  <node pkg="pr2_ik" type="pr2_ik_node" name="pr2_ik_node">    
   <param name="tip_name" value="l_wrist_roll_link" />     
   <param name="root_name" value="torso_lift_link" />  
  </node>
</launch>
